docs/content/how-do-i-write-policies.md

```diff
@@ -276,26 +282,27 @@ A simple example is a regex to match a valid Rego variable. With a regular strin
 
 Composite values define collections. In simple cases, composite values can be treated as constants like [Scalar Values](#scalar-values):
 
-```ruby
-cube := {"width": 3, "height": 4, "depth": 5}
+```live:eg/cube:module
+cube = {"width": 3, "height": 4, "depth": 5}
 ```
 
 The result:
 
-```ruby
-> cube.width
-3
+```live:eg/cube:query:merge_down
+cube.width
+```
+```live:eg/cube:output
 ```
 
 Composite values can also be defined in terms of [Variables](#variables) or [References](#references). For example:
 
-```ruby
-> a := 42; b := false; c := null; d := {"a": a, "x": [b, c]}
-+----+-------+------+---------------------------+
-| a  |   b   |  c   |             d             |
-+----+-------+------+---------------------------+
-| 42 | false | null | {"a":42,"x":[false,null]} |
-+----+-------+------+---------------------------+
+```live:eg/composite_variables:module:merge_down
```

Comment:
Does this have to be a module? Could it not just be the original query?

Reply:
Done; changes the displayed output to a pretty table.